src/test/java/com/microsoft/sqlserver/jdbc/bvt/bvtTest.java

```diff
@@ -1,678 +1,587 @@
+    // CONCUR_READ_ONLY, executeQuery
+    // verify cursor by using next and verify data
+    ///////////////////////////////////////////////////////////////////
+    @Test
+    public void testStmtSS_SEVER_CURSOR_FORWARD_ONLY() throws SQLException {
+
+        try {
+            int TYPE_SS_SEVER_CURSOR_FORWARD_ONLY = 2004;
+            int CONCUR_READ_ONLY = 1008;
+            conn = new DBConnection(connectionString);
+            stmt = conn.createStatement(TYPE_SS_SEVER_CURSOR_FORWARD_ONLY, CONCUR_READ_ONLY);
+
+            String query = "SELECT * FROM " + table1.getEscapedTableName();
+
+            rs = stmt.executeQuery(query);
+
+            // Verify resultset behavior
+            rs.next();
+            rs.verify(table1);
+        }
+        finally {
+            terminateVariation();
+        }
+
+    }
+
+    ///////////////////////////////////////////////////////////////////
+    // Create a preparedstatement, call close
+    ///////////////////////////////////////////////////////////////////
+    @Test
+    public void testCreatepreparedStatement() throws SQLException {
+
+        try {
+            conn = new DBConnection(connectionString);
+            String colName = table1.getColumnName(7);
+            String value = DBResultSet.getRow(table1, 0).get(7).toString();
+
+            String query = "SELECT * from " + table1.getEscapedTableName() + " where [" + colName + "] = ? ";
+
+            pstmt = conn.prepareStatement(query);
+            pstmt.setObject(1, new BigDecimal(value));
+           
+            rs = pstmt.executeQuery();
+            rs.verify(table1);
+        }
+        finally {
+            terminateVariation();
+        }
+    }
+
+    ///////////////////////////////////////////////////////////////////
+    // Verify resultset using ResultSetMetaData
+    ///////////////////////////////////////////////////////////////////
+    @Test
+    public void testResultSet() throws SQLException {
+
+        try {
+            conn = new DBConnection(connectionString);
+            stmt = conn.createStatement();
+
+            String query = "SELECT * FROM " + table1.getEscapedTableName();
+            rs = stmt.executeQuery(query);
+
+            // verify resultSet
+            rs.verify(table1);
+        }
+        finally {
+            terminateVariation();
+        }
+    }
+
+    ///////////////////////////////////////////////////////////////////
+    // Verify resultset and close resultSet
+    ///////////////////////////////////////////////////////////////////
+    @Test
+    public void testResultSetAndClose() throws SQLException {
+
+        try {
+            conn = new DBConnection(connectionString);
+            stmt = conn.createStatement();
+
+            String query = "SELECT * FROM " + table1.getEscapedTableName();
+            rs = stmt.executeQuery(query);
+            rs.currentTable = table1;
+
+            rs.verify(table1);
+            stmt.close();
+        }
```

Comment:
we should verify if the stmt/rs is closed here else this test will be same as the previous one.